_posts/2016-12-05-moore-report.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+subheadline: "Support"
+title: "A Year in Review: Annual Moore Progress Report"
+teaser: ""
+header:
+  image_fullwidth: "light-blue-wood-texture.jpg"
+categories:
+  - blog
+comments: true
+show_meta: true
+authors: ["Tracy Teal"]
+---
+
+
+A year ago we were fortunate enough to receive a grant from the 
+[Gordon and Betty Moore Foundation](https://www.moore.org/article-detail?newsUrlName=the-gordon-and-betty-moore-foundation-selects-awardees-for-$21-million-in-grants-to-stimulate-data-driven-discovery) 
+Data-Driven Discovery initiative to support Data Carpentry to develop and deliver data skills training to researchers in the life and 
+physical sciences. [See our original proposal here.](https://figshare.com/articles/Data_Carpentry_Moore_Foundation_Grant_Proposal/1507552)
+Thanks to the Moore Foundation, this support has allowed us to run more workshops, develop new materials, 
+develop infrastructure and advocate for the importance of data skills training for more effective and reproducible research.  
+
+
+We wanted to share the outputs and outcomes of Data Carpentry activities over this last year from our annual report to the Moore 
+Foundation.  
+
+
+- [Annual Report](https://github.com/datacarpentry/practices/blob/datacarpentry/datacarpentry/Moore_DataCarpentry_progress_report.pdf)  
+- [Output and Events - Excel file](https://github.com/datacarpentry/practices/blob/datacarpentry/datacarpentry/Data-Carpentry_Practices-Data-Collection.xlsx?raw=true)  
+
+
+
+### Some highlights from the last year  
+
+#### Workshops  
+
+- Since August, 2015 we have run 58 workshops in ten countries, serving approximately 1,100 learners  
+- We have over 600 completed surveys that show the demographics of our audience as well as workshop outcomes.
```

Comment:
We've now run 93 workshops for ~1700 learners. We can update the numbers, or put Aug 2015 to Sept 2016.